CMakeLists.txt

```diff
@@ -253,6 +253,14 @@ camp source repository")
       add_subdirectory(tpl/camp)
     endif()
   endif (EXTERNAL_CAMP_SOURCE_DIR)
+
+  if (RAJA_ENABLE_TARGET_OPENMP)
```

Comment:
@davidbeckingsale are you OK with these changes?

Reply:
Yup, seems reasonable - matches what we do in Umpire to ensure correct defines are passed to camp.